.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  architect: giantswarm/architect@0.1.2
+
+workflows:
+  package-and-push-chart-on-tag:
```

Comment:
Just curious, Don't we need to push on docker first?

Reply:
I don't think so as there is no image. The chart will use retagged versions of the Kong images.